cli/waiter/action.py

```diff
@@ -287,3 +287,67 @@ def process_ping_request(clusters, token_name_or_service_id, is_service_id, time
                 success = True
         overall_success = overall_success and success
     return overall_success
+
+
+def kill_instance_on_cluster(cluster, instance_id, timeout_seconds):
+    """Send sigkill request to the specific instance"""
+    cluster_name = cluster['name']
+    http_util.set_retries(0)
+    try:
+        print(f'Sending sigkill request to instance {terminal.bold(instance_id)} in {terminal.bold(cluster_name)}...')
+        params = {'timeout': timeout_seconds * 1000}
+        resp = http_util.delete(cluster, f'/apps/{instance_id}/signal', params=params, read_timeout=timeout_seconds)
+        
+        logging.debug(f'Response status code: {resp.status_code}')
+        if resp.status_code == 200:
+                    success = resp.json().get("kill-response").get('success')
+                    if success:
+                        print(f'Successfully killed {instance_id} in {cluster_name}.')
+                        return True
+                    else:
+                        print(f'Was not able to kill {instance_id} in {cluster_name}. ')
+                        return False
+        else:
+            print_error(response_message(resp.json()))
```

Comment:
we probably want to log the response and only print a field from the response defaulting to n error message if it is missing